worlds/generic/docs/advanced_settings_en.md

```diff
@@ -225,9 +235,12 @@ triggers:
 * `start_location_hints` gives us a starting hint for the `Spike Cave` location available at the beginning of the
   multiworld that can be used for no cost.
 * `exclude_locations` forces a not important item to be placed on the `Cave 45` location.
-* `item_links` causes all players with the same `item_links` settings to share a `Fire Rod` and `Ice Rod`. Extra 
-  `Rupee (1)` are put in the item pool instead of additional Rods.
-
+* `item_links` 
+  * For `A Link to the Past` all players in the `rods` item link group will share their fire and ice rods and the player
+items will be replaced with single rupees.
+  * For `Timespinner` all players in the `TSAll` item link group will share their entire item pool and the`Twin Pyramid 
+Key` and `Timespinner Wheel` will be forced among the worlds of those in the group. The `null` replacement item instead
+of forcing a filler item will allow the generator to fill the item.
```

Comment:
> The `null` replacement item instead of forcing a filler item will allow the generator to fill the item.

The last sentence here doesn't make sense to me. Please elaborate or re-write.

Reply:
I worded it a bit different to fix it